zerver/webhooks/newrelic/doc.md

```diff
@@ -1,27 +1,115 @@
+# New Relic ClickUp integration
```

Comment:
ClickUp?

I think this should be `# Zulip New Relic integration`.

Reply:
whoopsie :sweat_smile: